src/main/java/org/jenkinsci/plugins/workflow/steps/RetryStep.java

```diff
@@ -83,6 +85,16 @@ public Set<? extends Class<?>> getRequiredContext() {
             return Collections.singleton(TaskListener.class);
         }
 
+        public FormValidation doCheckCount(@QueryParameter int count) {
+            if (count < 1) {
+                return FormValidation.error("Count must be positive.");
```

Comment:
We could. For now I do not see a strong need since I was only concerned that users might mistakenly run `retry (1)` thinking that meant “please retry once after a failure”.